src/api_server/swagger/firecracker.yaml

```diff
@@ -399,6 +399,81 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /snapshot/create:
+    put:
+      summary: Creates a full or diff snapshot. Post-boot only.
+      description:
+        Creates a snapshot of the microVM state.
+      operationId: createSnapshot
+      parameters:
+        - name: body
+          in: body
+          description: The configuration used for creating a snaphot.
+          required: true
+          schema:
+            $ref: "#/definitions/CreateSnapshotParams"
+      responses:
+        501:
+          description: Not implemented yet
+        400:
+          description: Snapshot cannot be created due to bad input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Internal server error
+          schema:
+            $ref: "#/definitions/Error"
+
+  /snapshot/load:
+    put:
+      summary: Loads a snapshot. Pre-boot only.
+      description:
+        Loads the microVM state from a snapshot.
+      operationId: loadSnapshot
+      parameters:
+        - name: body
+          in: body
+          description: The configuration used for loading a snaphot.
+          required: true
+          schema:
+            $ref: "#/definitions/LoadSnapshotParams"
+      responses:
+        501:
+          description: Not implemented yet
+        400:
+          description: Snapshot cannot be loaded due to bad input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Internal server error
+          schema:
+            $ref: "#/definitions/Error"
+
+  /vm:
+    patch:
+      summary: Updates the microVM state.
+      description:
+        Sets the desired state for the microVM.
```

Comment:
Please make it more explicit: Pause/Resume

Reply:
Done.